lib/xcodeproj/project/object/helpers/file_references_factory.rb

```diff
@@ -235,6 +236,28 @@ def find_products_group_ref(group, should_create = false)
             product_group_ref
           end
 
+          # Find the remote info name for a new sub project.
+          #
+          # @param  [PBXFileReference] ref
+          #         The file reference to compare.
+          #
+          # @param  [Project] project
+          #         The project to which to find the target.
+          #
+          # @note   If the target is found, then return the name of target,
+          #         otherwise, the default value 'Subproject' will be returned
+          #
+          # @return [String] The remote info name.
+          #
+          def find_remote_info_name_for_product_reference(ref, project)
+            project.native_targets.each do |target|
```

Comment:
this can be:
```ruby
def find_remote_info_name_for_product_reference(ref, project)
  project.native_targets.find { |t| t.product_reference == ref } || `'Subproject'
end
```